### DISCUSSION TITLE
Welcome to english-amharic-tech-terms Discussions!

### DISCUSSION THREAD

**Discussion by zemachfm** [zemachfm/english-amharic-tech-terms#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fesseha1** [comment]
Please explain to me why do we need to create a technology term in Amharic ? For example: TV is TV in all languages (Derived from a technical term Tele - Vision ...I have on my website a technology translation book from 30 years ago that comes up with a term " ነሲቡ አስታውሴ" for RAM Random Access Memory i.e ) But I am not convinced it is worth it .. But I am willing to listen .

**Reply by henoktsegaye** [reply]
Good point, I have tried to point out some of my reasons why we need to translate below

**Comment by henoktsegaye** [comment]
Well, thanks for bringing this point because it's a great discussion point., I know agreeing on this one will make the whole process fast, or maybe we might not have a process after this discussion but I am here to persuade you we need a process, a process to start making our languages techy, if not our languages, culture and our influence is at steak :(

So I am gonna take a deep dive into this topic, so bear with me please   🙏🏾

Have you ever wondered why we learn in English after grade 7 ( that is in my case, this might be lower or higher based on where you learn ), I feel like that is something to really think about because learning English is not easy besides we have to learn English, then learn other subjects in English! That was a hustle, actually, a lot of people are failing because of this, I can say this from experience. People have to learn the language that is not used every day which is only in one class then use that knowledge to learn other important stuff based on their choice of field. well if you are not exposed to English in other forms like movies or music, my opinion is you are destined to fail, now you might be asking why are you talking about this? all right I am gonna get to the point, there are a lot of people that don't know or know a little about English, those people are struggling I can tell you from experience, How do you think a person that doesn't know English use a phone? I can answer that, for that person, it's purely a step nothing makes sense for them, Do you think this is the same for people that speak English? it definitely is not, they know what it means and they can navigate fairly easily.

Tech is for everyone we can't limit tech to people that know English, a country like ours is in need of tech and tech should be used everywhere by everyone, so when we bring tech to everyone, what language do you think we should use? 

now, this is all about language, and that is just the first argument there are a couple in my pocket 😀  but you might ask why to translate terms in the first place? if they are English let them stay that way! If we agree with the first argument I agree with this to some extent here is why we [here at zemach fm] are not only translating nouns we are translating verbs as well eg. delete, what do you call that in Amharic ( would you still call it "delete" ?), now this is a direct translation but when we do translate this word we have to make sure this verb is in context with tech. CONTEXT is Key

now in your example TV might not need a translation because we are already used to the word itself, there are a couple of things to consider when translating a word, another example is a fridge, this might be up for discussion but those are already well-used words might not be easy to translate because new words always don't make sense. but things like "speaker" are given a name "ድምጽ ማጉያ" which makes total sense and that can be used in Amharic terms and is self-explanatory which is helpful, which would be easier for the Amharic audience and it will make the language grow as well

which takes me to my second big argument,
 WHY
 do we still use Ethiopian languages ? in this case I am going to pick Amharic.  I remember asking myself this question "Why do we continue using Amharic ?"
my question was based on
 1. we can learn well in English and English have everything we can learn about ( usually the case )
2. people in college learn in English but they work in Amharic, that didn't make sense to me.
and I remember I asked myself why even bother using Amharic, well that question was far too deep, well as you can tell using an Ethiopian language for Ethiopians have much more meaning than just using it at work or school, it's much more sophisticated? our languages have in them our identity how we describe things describe us, how we communicate is embedded in our languages, it's just so much more, now losing a language for a country and adopting another foreign language is monumental not in a good way, 
now if you agree with that, ask yourself what if languages are not usable though?
what I mean by un-usable is, when a language is weak to express an idea without borrowing from some other languages 20 times in a statement, well that language is weak, you might even ask what is the point of using this language in the first place, besides if the language is weak, its obvious its going to die and we don't want that because languages are not only words, besides Ethiopia has one of the 18 languages that have its own alphabet, so if that language dies its going to be sad for the world too.

But here is a fact 😁 if the language can't be usable we are not going to use it, so a language needs to create new vocabularies to cope with the fast-changing world which is the second and most important reason why we translate words to Ethiopian languages
